src/main/java/com/iexec/core/replicate/ReplicateSupplyService.java

```diff
@@ -531,12 +532,13 @@ private TaskAbortCause getTaskAbortCause(Task task) {
 
     // region purge locks
     @Override
-    public boolean purgeTask(String chainTaskId) {
+    public boolean purgeTask(final String chainTaskId) {
```

Comment:
```suggestion
    public boolean purgeTask(final String chainTaskId) {
        log.debug("purgeTask [chainTaskId: {}]", chainTaskId);
```

Reply:
resolved in : [a35058a](https://github.com/iExecBlockchainComputing/iexec-core/pull/724/commits/a35058a652778f31cfebf678aab460f24ca556b8)